### ISSUE TITLE
Display error messages in plain English that catalogers can understand.

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#2190]
Error messages are either too technical or don't have any relevant meaning for catalogers. Could we change them to something that is in plain English or provide a way to help the cataloger.

**Comment by michelleif** [comment]
@astridu do you have a list of specific error messages that catalogers don't understand?

**Comment by astridu** [comment]
I do not, but I can start collecting them.

**Comment by michelleif** [comment]
we do have 3 open tix for specific error messages (#2457, #2861, #2860).

how about we close this ticket, with the understanding that if you come across reports of unclear error messages, we will ticket separately.
